idd/Energy+.idd.in

```diff
@@ -1,5 +1,5 @@
-!IDD_Version ${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}.${CMAKE_VERSION_PATCH}
-!IDD_BUILD ${CMAKE_VERSION_BUILD}
+!IDD_Version 8.3.3
+!IDD_BUILD Unknown
```

Comment:
@jasondegraw Thanks for pointing this out. I reverted it back in Energy+.idd.in on my local machine and commit it soon.